helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -167,6 +167,30 @@ extraAttributes:
     # - k8s-app
     # - release
 
+  # Labels that will be collected from k8s namespaces (in case they are set)
+  # and added  as extra attributes to the telemetry in the following format:
+  # k8s.namespace.labels.<label_name>: <label_value>
+  namespaceLabels: []
+#    - dev
+#    - staging
+#    - production
+
+  # Annotations that will be collected from k8s pods (in case they are set)
+  # and added  as extra attributes to the telemetry in the following format:
+  # k8s.pod.annotations.<label_name>: <label_value>
+  podAnnotations: []
+    # - app
```

Comment:
these examples doesn't match with commonly used annotations. let's not have them.